zproject/settings.py

```diff
@@ -965,6 +965,14 @@ def get_secret(key):
     POPULATE_PROFILE_VIA_LDAP = 'zproject.backends.ZulipLDAPAuthBackend' in AUTHENTICATION_BACKENDS or POPULATE_PROFILE_VIA_LDAP
 
 ########################################################################
+# GITHUB AUTHENTICATION SETTINGS
+########################################################################
+SOCIAL_AUTH_USER_MODEL = AUTH_USER_MODEL
```

Comment:
https://github.com/omab/python-social-auth/blob/master/social/apps/django_app/default/models.py seems to suggest `SOCIAL_AUTH_USER_MODEL` defaults to `AUTH_USER_MODEL` anyway; is this line needed?